src/App.js

```diff
@@ -1,25 +1,34 @@
-import logo from './logo.svg';
 import './App.css';
+import React, { useEffect } from 'react';
+import { useAppSelector, useAppDispatch } from './redux/hooks';
+import { fetchGreeting } from './redux/slices/greetingSlice';
+
+const App = () => {
+  const dispatch = useAppDispatch();
+  const greeting = useAppSelector((state) => state.greeting.value);
+  const loading = useAppSelector((state) => state.greeting.loading);
+  const error = useAppSelector((state) => state.greeting.error);
+
+  useEffect(() => {
+    dispatch(fetchGreeting());
+  }, []);
```

Comment:
Nice job so far😉. As a requirement, you are to create a Greeting component that will display a greeting. You are also required to set it up as a route in your App component using react-router😉. Please do well to make adjustments👍🏼. Kindly do well to stick to the instructions as shown in the image below😉.


<img width="797" alt="image" src="https://github.com/bhushan354/hello-rails-front-end/assets/64643995/e362faa1-9084-4faf-a812-500447bf20e2">